packages/cli/src/commands/site-configs.ts

```diff
@@ -10,6 +10,7 @@ export const injectSiteConfigsCommand = new Command("inject-site-configs")
     .description("Inject site-configs into a file")
     .requiredOption("-i, --in-file <file>", "The filename of a template file to inject.")
     .requiredOption("-o, --out-file <file>", "Write the injected template to a file.")
+    .option("-d --dotenv", "dotenv compatibility") // https://github.com/motdotla/dotenv/issues/521#issuecomment-999016064
```

Comment:
Shouldn't this be something we do all the time since we're using dotenv anyway?

Reply:
Okay, but could we perform the workaround anyway, even if it does nothing in deployed environments? I'd like to keep the options of the command at a minimum.

Reply:
Discussed it internally and tried alternative solutions, but working around the issue here seems like the easiest solution.